lib/vquic/vquic.c

```diff
@@ -385,7 +390,7 @@ static CURLcode recvmmsg_packets(struct Curl_cfilter *cf,
       mmsg[i].msg_hdr.msg_name = &remote_addr[i];
       mmsg[i].msg_hdr.msg_namelen = sizeof(remote_addr[i]);
       mmsg[i].msg_hdr.msg_control = &msg_ctrl[i];
```

Comment:
The offset needs to be adjusted with CMSG_SPACE boundary:

```suggestion
      mmsg[i].msg_hdr.msg_control = &msg_ctrl[i * CMSG_SPACE(sizeof(int))];
```

Reply:
Thanks, just pushed this change.